explorer/src/alto_types/alto_types.js

```diff
@@ -84,8 +84,9 @@ export function parse_block(bytes) {
  * @param {number} participants
  * @returns {number}
  */
-export function leader_index(signature, participants) {
-    const ptr0 = passArray8ToWasm0(signature, wasm.__wbindgen_malloc);
+export function leader_index(seed, participants) {
+    // FIXME
```

Comment:
We need to pass the `SeedJs` value back to WASM. Previously we were passing just the signature bytes.